openpype/hosts/blender/plugins/load/load_animation.py

```diff
@@ -9,64 +9,17 @@
 
 class AnimationLoader(plugin.AssetLoader):
     """Load animations from a .blend file."""
+    families = ["animation"]
+    representations = ["blend"]
```

Comment:
Don't set `representations `+ `families ` with this AbstractLoader cause it will appear in the loader interface.
![capture](https://user-images.githubusercontent.com/13371155/216343990-fc36cdb0-13da-4511-9da4-8031e850d21a.png)

Reply:
Done, tested and it is listed only when `representations =` is filled.